.github/workflows/iroha2-dev-nightly.yml

```diff
@@ -3,17 +3,25 @@ name: I2::Dev::Nightly::Publish
 on: workflow_dispatch
 
 env:
+  RUST_TOOLCHAIN: nightly-2024-04-18
   DOCKER_COMPOSE_PATH: defaults
 
 jobs:
   build_executor:
-    runs-on: ubuntu-latest
-    container:
-      image: hyperledger/iroha2-ci:nightly-2024-04-18
```

Comment:
We don't need `Dockerfile.build` then

Reply:
There is at least one job left that still uses the self-hosted runner. I'll see if I can move it too